assets/js/base/utils/render-frontend.tsx

```diff
@@ -0,0 +1,206 @@
+/**
+ * External dependencies
+ */
+import { render, Suspense } from '@wordpress/element';
+import BlockErrorBoundary from '@woocommerce/base-components/block-error-boundary';
+
+// Some blocks take care of rendering their inner blocks automatically. For
+// example, the empty cart. In those cases, we don't want to trigger the render
+// function of inner components on load. Instead, the wrapper block can trigger
+// the event `wc-blocks_render_blocks_frontend` to render its inner blocks.
+const selectorsToSkipOnLoad = [ '.wp-block-woocommerce-cart' ];
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type blockType = ( props: any ) => JSX.Element;
```

Comment:
I couldn't find any way to replace this `any` with something more specific. Each block passed as a param to this function can have different required props and I couldn't find any way to let TypeScript ignore them. :confused:

Reply:
I tried with `Record<string, unknown>` and  I didn't see any error. I think that it can be a good fit, because we are sure that we will pass an object

Reply:
I think that for the name type it is better if we use PascalCase. So in this case `BlockType`. What do you think?

Reply:
> I tried with `Record<string, unknown>` and  I didn't see any error. I think that it can be a good fit, because we are sure that we will pass an object

The error doesn't appear right there, but in consumers of `renderFrontend` and `renderBlock` (ie, [here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/95ac903535b45674058148dacd255edc302fba39/assets/js/blocks/cart-checkout/mini-cart/component-frontend.tsx#L30)).

> I think that for the name type it is better if we use PascalCase. So in this case `BlockType`. What do you think?

Oh, right, I have been doing it wrong for a long time. :man_facepalming: I updated it in this file and a couple more in 3029677cfba7ac725a27111b2cfae67a81b3fc4d.